configs/stashcache.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
Can you rename this file to something other than `stashcache.service`  Maybe `stashcache-reporter.service`